project/make/cross

```diff
@@ -28,6 +30,21 @@ for platform in $DOCKER_CROSSPLATFORMS; do
 			export LDFLAGS_STATIC_DOCKER="" # we just need a simple client for these platforms
 			export BUILDFLAGS=( "${ORIG_BUILDFLAGS[@]/ daemon/}" ) # remove the "daemon" build tag from platforms that aren't supported
 		fi
+		if [ $GOARCH = "arm" ] && [ $GOOS = "linux" ]
+		then
+			export CC=arm-linux-gnueabi-gcc
+			export LD_FLAGS="-extld=arm-linux-gnueabi-gcc"
+			export CGO_ENABLED=1
+			export LIBRARY_PATH=$LIBRARY_PATH:/usr/arm-linux-gnueabi/lib
```

Comment:
I'm pretty sure these will cause problems for users who are compiling outside the build containers, as I described in my earlier comment on this PR (`./hack/make.sh dynbinary` natively on the target architecture).